### PR TITLE
feat: [SRM-14997]: Update SRMCustomMicroFrontendProps and Move account level slo listing to MFE

### PR DESCRIPTION
**Pull request by deepesh-ui**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/RouteDestinations.tsx [hunk 1]
```diff
@@ -671,22 +685,45 @@ export const SRMMFERoutes: React.FC = () => {
...
-        </RouteWithLayout>
+        <>
+          <RouteWithLayout
+            exact
+            sidebarProps={CVSideNavProps}
+            path={routes.toCVSLOs({ ...accountPathProps, ...projectPathProps, ...cvModuleParams })}
+          >
+            <CVSLOsListingPage />
+          </RouteWithLayout>
+          <RouteWithLayout exact sidebarProps={CVSideNavProps} path={routes.toAccountCVSLOs({ ...accountPathProps })}>
```
Comment (praneshg239) [inline src/modules/85-cv/RouteDestinations.tsx:723]: why we are having two `RouteWithLayout` here? Won't it render two page like UI in same screen?

Comment (deepesh-ui) [reply src/modules/85-cv/RouteDestinations.tsx:723]: renders the one with exact match only

Comment (praneshg239) [reply src/modules/85-cv/RouteDestinations.tsx:723]: understood, I missed it

File: src/microfrontends/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/microfrontends",
-  "version": "1.62.0",
+  "version": "1.60.1",
```
Comment (praneshg239) [inline src/microfrontends/package.json:3]: do we need to reduce the version?

**Review by praneshg239 — DISMISSED** [review]

File: src/microfrontends/package.json (hunk 2 above)
Comment (knagurski) [reply src/microfrontends/package.json:3]: This looks like a blocker to me. We can't go back in version numbers.

Comment (deepesh-ui) [reply src/microfrontends/package.json]: @praneshg239 @knagurski   My bad, Updated version

File: src/microfrontends/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/microfrontends",
-  "version": "1.62.0",
+  "version": "1.62.1",
```
Comment (knagurski) [inline src/microfrontends/package.json:3]: Is this a bug fix? It looks like the ticket is for a feature.
```suggestion
  "version": "1.63.0",
```

Comment (deepesh-ui) [reply src/microfrontends/package.json]: @knagurski  Its feature , we are migrating srm module to micro frontend ,

**Review by knagurski — APPROVED** [review]
👍🏻

**Review by praneshg239 — APPROVED** [review]

**Pull request merged**